towhee/functional/mixins/entity_mixin.py

```diff
@@ -199,6 +199,27 @@ def inner(x):
 
         return self.factory(map(inner, self._iterable))
 
+    def select(self, *args):
+        """
+        Get the list from operator Output.
+
+        Examples:
+
+        >>> from towhee import Entity
+        >>> from towhee import DataCollection
+        >>> (
+        ...     DataCollection([Entity(a=i, b=i, c=i) for i in range(5)])
+        ...         .select('a', 'b')
+        ...         .as_raw()
+        ...         .to_list()
+        ... )
+        [(0, 0), (1, 1), (2, 2), (3, 3), (4, 4)]
+        """
```

Comment:
There also needs a code example to explain the behavior without `as_raw()`

Reply:
good idea